cumulusci/tasks/apex/testrunner.py

```diff
@@ -283,13 +303,81 @@ def _get_test_class_query(self):
         return query
 
     def _get_test_classes(self):
+        # If test_suite_names is provided, execute only tests that are a part of the list of test suites provided.
+        if (
+            self.options["test_suite_names"] is not None
+            and len(self.options["test_suite_names"]) > 0
+        ):
+            test_classes_from_test_suite_names = (
```

Comment:
This early return makes the logic a bit harder to follow in my opinion. What do you think about pulling the code below in this function out into a separate function, so that you can have a simple if/else here?

Reply:
That's a fair point. We have written a separate function that fetches all the test classes from the org. The aforementioned function is run when the else block is executed. Thanks for pointing it out.